sentry-android-fragment/src/main/java/io/sentry/android/fragment/internal/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -14,24 +14,16 @@ import io.sentry.ISpan
 import io.sentry.SentryLevel.INFO
 import io.sentry.SpanStatus
 import io.sentry.TypeCheckHint.ANDROID_FRAGMENT
+import io.sentry.android.fragment.FragmentLifecycleState
 import java.util.WeakHashMap
 
 @Suppress("TooManyFunctions")
-class SentryFragmentLifecycleCallbacks(
+internal class SentryFragmentLifecycleCallbacks(
```

Comment:
If we are going to revert the `internal` part, we should also keep the old constructor, similar how we did it with `FragmentLifecycleIntegration`